storage/innobase/include/dict0mem.h

```diff
@@ -1796,6 +1803,31 @@ struct dict_foreign_add_to_referenced_table {
 	}
 };
 
+/** Request for lazy creation of the analyze index mutex of a given table.
+@param[in,out]	table	table whose mutex is to be created. */
```

Comment:
This is the correct format for InnoDB. Can you please use the same for rest of the new code?